packages/model-viewer/src/model-viewer-base.ts

```diff
@@ -110,6 +110,8 @@ export interface RendererInterface {
 export default class ModelViewerElementBase extends UpdatingElement {
   protected static[$template]: HTMLTemplateElement|void;
 
+  public static powerPreference:string = 'high-perfromance';
```

Comment:
Regarding your question; perhaps a static set method that calls `resetSingleton()` if the value has changed? If they set it before the element is loaded, it should still start up correctly without a reset (though I agree our automated test won't be covering that situation).